src/screens/FeedScreen/index.js

```diff
@@ -39,25 +40,27 @@ const FeedScreen = (props) => {
     bottom,
     loading,
     myProfile,
-    nextTargetFeed,
-    postOffset,
     searchHeight,
     showNavbar,
+    timer,
+    viewPostTimeIndex,
 
     checkCacheFeed,
+    fetchNextFeeds,
     getDataFeeds,
     handleScroll,
-    isSamePostViewed,
     onBlockCompleted,
     onDeleteBlockedPostCompleted,
     saveSearchHeight,
     sendViewPostTime,
     setDownVote,
     setIsLastPage,
     setShowNavbar,
-    setUpVote,
-    updateViewPostTime
+    setUpVote
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Several variables and functions have been added and removed from the destructuring of `useCoreFeed()`. Make sure that all new additions are properly defined in the `useCoreFeed` hook and that any removed items are no longer needed in this component.



<!-- This is an auto-generated comment by OSS CodeRabbit -->